### DISCUSSION TITLE
Window object like in browsers

### DISCUSSION THREAD

**Discussion by okarpov** [sebastienros/jint#1255] [Ideas]
I'm sure it will definitely be the huge improvement with a lot of demand to have window object like browsers have.

What do you think?

**Comment by lahma** [comment]
You might want to have a look at [AngleSharp.Js](https://github.com/AngleSharp/AngleSharp.Js) for ideas, it's a project providing some of the bindings.

**Reply by okarpov** [reply]
yes, but i meant comprehensive Window and Dom and not just a simple stubs

**Reply by lahma** [reply]
So would you like to contribute such feature?

**Reply by okarpov** [reply]
i wish I could, but unfortunately my contribution will take ages.
we have a russian war, and i need to feed my wife and daughter 
so i will have no time in the nearest future

sorry

**Reply by lahma** [reply]
I understand, stay safe. Slava Ukraini!

**Comment by okarpov** [comment]
So may be there is a possibillity to compile and run the https://github.com/jsdom/jsdom ?
or any other way to integrate jsdom into this project? (i'm not a js/node developer)

**Reply by lahma** [reply]
That looks a bit complex and might rely on Node APIs. Also that would require NPM dependency loading.
